python/lsst/analysis/tools/actions/plot/plotUtils.py

```diff
@@ -42,55 +42,14 @@
 null_formatter = matplotlib.ticker.NullFormatter()
 
 
-def parsePlotInfo(dataId, runName, tableName, bands, plotName, SN):
```

Comment:
This is being moved from `plotUtils` into `diffimTaskDetectorVisitSpatiallySampledAnalysis`?

Reply:
Ah, I suppose this is being removed to defer to the main `parsePlotInfo` inside `analysis/tools/interfaces/_task.py` instead?

Reply:
No, this is a function that's not called from anywhere. Instead, its namesake which a method of `AnalysisPipelineTask` is the one that's used everywhere. I'm removing it to avoid any confusion.